diff --git a/node_modules/epub2/lib/epub.js b/node_modules/epub2/lib/epub.js
index 729208b..b5dd4ac 100644
--- a/node_modules/epub2/lib/epub.js
+++ b/node_modules/epub2/lib/epub.js
@@ -717,7 +717,7 @@ class EPub extends events_1.EventEmitter {
      **/
     getChapterRaw(chapterId, callback) {
         if (this.manifest[chapterId]) {
-            if (!(this.manifest[chapterId]['media-type'] == "application/xhtml+xml" || this.manifest[chapterId]['media-type'] == "image/svg+xml")) {
+            if (!(this.manifest[chapterId]['media-type'] == "application/xhtml+xml" || this.manifest[chapterId]['media-type'] == "image/svg+xml" || this.manifest[chapterId]['media-type'] == "text/html")) {
                 return callback(new Error(`Invalid mime type for chapter "${chapterId}" ${this.manifest[chapterId]['media-type']}`));
             }
             this.zip.readFile(this.manifest[chapterId].href, (function (err, data) {
